exercises/dominoes/.meta/src/reference/java/Dominoes.java

```diff
@@ -0,0 +1,69 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+
+class Dominoes {
+	
+	ArrayList<Domino> formChain(ArrayList<Domino> inputDominoes) {
```

Comment:
Do you need the input to be an `ArrayList`? I would keep it just a general `List` to make the tests a bit neater and to give the users a bit more freedom in their implementation 🙂 

And you can change the return type to just `List` as well 🙂